assignments/arrays.js

```diff
@@ -62,35 +64,81 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // }
 
 // ==== Challenge 1 ====
-// The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car 
+// has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
 
+// const carYear = inventory[33-1].car_year;
+// const carMake = inventory[33-1].car_make;
+// const carModel = inventory[33-1].car_model;
+// const carId = inventory[33-1]['id'];
+// console.log(`Car ${carId} is a ${carYear} ${carMake} ${carModel}`);
 
+// Not sure if this is what I was supposed to do but there you have it. 
+const carFinder = function(inventory, carId) {
```

Comment:
You overthought this a bit, but I appreciate the  enthusiasm and resourcefulness. Well done on this!